src/frontend/Sider/Behandling/Venstremeny/Oppsummering/OppsummeringSøknad.tsx

```diff
@@ -24,6 +25,7 @@ const OppsummeringSøknad: React.FC = () => {
                     </BodyShort>
                 </InfoSeksjon>
             )}
+            <RevurderingTag forrigeBehandlingId={behandling.forrigeBehandlingId} />
```

Comment:
For å vite om det er en revurdering så burde vi nog bruke `type` på behandling.
I et tilfelle der man først har et avslag og sen en ny behandling, så kommer `forrigeBehandlingId` være null.
`type` sier om det gjelder en førstegangsbehandling eller revurdering

`forrigeBehandlingId` har kun verdi når en tidligere behandling har blitt iverksatt - og då sendt utbetalingsperioder til økonomi